src/main/java/calculator/StringAddCalculator.java

```diff
@@ -0,0 +1,46 @@
+package calculator;
+
+import java.util.Arrays;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+
+    /**
+     * 숫자 하나를 문자열로 입력할 경우 해당 숫자를 반환한다.(예 : “1”)
+     * 숫자 두개를 컴마(,) 구분자로 입력할 경우 두 숫자의 합을 반환한다.(예 : “1,2”)
+     * 구분자를 컴마(,) 이외에 콜론(:)을 사용할 수 있다. (예 : “1,2:3” => 6)
+     * 빈 문자열 또는 null 값을 입력할 경우 0을 반환해야 한다.(예 : “” => 0, null => 0)
+     *
+     * @param text
+     * @return int
+     */
+    public static int splitAndSum(String text) {
+        if (null == text || text.isEmpty()) {
+            return 0;
+        }
+
+        return Arrays.stream(splitText(text)).mapToInt(StringAddCalculator::getPositiveInteger).sum();
```

Comment:
이부분 관련하여 질문이 있는데요. 

간결하게 한줄로 표현이 좋을까요?
아니면 아래처럼 메소드를 step별로 분리를하는게 좋을까요?

```suggestion
String[] numbers = splitText(text);
        return Arrays.stream(numbers).mapToInt(StringAddCalculator::getPositiveInteger).sum();
```